articles/portal/ptl-ref-release-notes.md

```diff
@@ -25,6 +25,46 @@ Bug fix
 
 Details: Fixed error when managing IPsec VPN configuration in Elevated.
 
+## Release 1.88.0
```

Comment:
Sorry, the release notes in Confluence are wrong, the version change was not actually customer facing, so this entry is probably not relevant.